backend/custom_models/migrations/0013_upload_background_tree_view_info.py

```diff
@@ -43,11 +43,11 @@ def upload_background_tree_view_info(apps, schema_editor):
     )
 
     # create item_types; do not use bulk_create since we need these objects later on
-    it_tree_view = item_type(type_desc="tree-view")
```

Comment:
I renamed them, because it was way too complicated to call this name in the api call because of the dashes. It is just simpler to call these with "_" in the api calls.
Yes, we'll need to roll back to 12 and then migrate again if you already migrated 13.